src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/IncompleteServiceReportITest.java

```diff
@@ -0,0 +1,179 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/mod/truncate.sql",
+    "/db/mod/reports/IncompleteServiceReportITest_typical.sql"
+})
+@SuppressWarnings("PMD.LawOfDemeter")
+class IncompleteServiceReportITest extends AbstractStandardReportControllerITest {
+    @Autowired
+    public IncompleteServiceReportITest(TestRestTemplate template) {
+        super(template, IncompleteServiceReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .reportType("IncompleteServiceReport")
+            .date(LocalDate.now())
+            .locCode("415")
+            .build());
+    }
+
+    @Test
+    @SuppressWarnings({
+        "PMD.JUnitTestsShouldIncludeAssert"//False positive
+    })
+    void positiveTypicalCourt() {
+        testBuilder()
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated)
+            .printResponse()
+            .assertEquals(getTypicalResponse());
+    }
+
+    @Test
+    @SuppressWarnings({
+        "PMD.JUnitTestsShouldIncludeAssert"//False positive
+    })
```

Comment:
This can go at the top of the class to save duplicating on each method (This happens for all of the testBuilder logic I still need to workout a way around this)

Reply:
done